console/controllers/MyController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace console\controllers;
+
+use Yii;
+use console\models\Person;
+use yii\console\Controller;
+use yii\db\Command;
+use yii\helpers\ArrayHelper;
+
+class MyController extends Controller
+{
+ 
+   public function actionIndex()
+    {
+
+$z = 0;
+while($z < 3) {
```

Comment:
Please use progressbar helper for displaying progress
https://gist.github.com/artemrogov/84b7ff6f68c20ffc0b54caa54ca053d7